### DISCUSSION TITLE
[Bug]: Android input forms cannot scroll down to submit

### DISCUSSION THREAD

**Discussion by Joeviocoe** [blakeblackshear/frigate#13687] [Report a Bug] [answered]
### Checklist

- [X] I have updated to the latest available Frigate version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [X] I have tried reproducing the issue in [incognito mode](https://www.computerworld.com/article/1719851/how-to-go-incognito-in-chrome-firefox-safari-and-edge.html) to rule out problems with any third party extensions or plugins I have installed.

### Describe the problem you are having

Input form overlays such as Add Camera Group or Export clip, anything with a text input, will be obscured after Android on-screen keyboard is used, resulting in the user not able to complete/apply/save entry.

Only workaround is to long press and paste from a clipboard to avoid using any on-screen keyboard in Android.

### Steps to reproduce

1. On Android phone running the Frigate UI through the Home Assistant companion app.
2. Click "+" button next to 'house icon' to create new camera group
3. Click "+" button next to "Camera Groups"
4. Click into the "Name" text field.
5. Type anything and click enter or back out of the on-screen keyboard
6. Form overlay will not return to full size and it is not possible to scroll down to reveal the Save or Cancel buttons
...

![Screenshot_20240912-083440_1](https://github.com/user-attachments/assets/a5d23e45-6478-4c55-a050-b11aef564165)

### Version

Beta 0.14.1

### In which browser(s) are you experiencing the issue with?

Home Assistant Companion for Android

### Frigate config file

```yaml
N/A
```

### docker-compose file or Docker CLI command

```yaml
N/A
```

### Relevant Frigate log output

```shell
N/A
```

### Relevant go2rtc log output

```shell
N/A
```

### Operating system

Debian

### Install method

HassOS Addon

### Network connection

Wireless

### Camera make and model

N/A

### Screenshots of the Frigate UI's System metrics pages

N/A

**Comment by NickM-27** [comment]
This is probably a bug with the HA app, I'm not able to reproduce in browser directly or in Frigate PWA

**Reply by Joeviocoe** [reply]
I tested in a few Android browsers.  

1. Firefox exhibits the same problem as the Home Assistant app.  When the keyboard is removed after typing text, the form overlay does not fully extend upwards, and hides most of the form, cannot scroll up or extend it.
2. Brave and Vanadium (Chromium based) do work better.  But there is still an issue.  The form overlay will  be fully extending with only a bit of the bottom cut off.  I can still click on half the Save or Cancel buttons.
![Screenshot_20240912-083355](https://github.com/user-attachments/assets/128f3817-555a-406f-b0c0-a8f0e0bd8f29)

What mobile browser are you testing for reproduction?

It is hard to pin it on the Home Assistant companion app team, as this type of form overlay looks to be animated and uses fancy JS.  I don't know of any other addon for HA that uses this kind of form, so they'll just punt it back to Frigate.  Especially since it is doing the same on Firefox.

**Reply by NickM-27** [reply]
I used chrome browser and chrome as pwa. Just tried Firefox and I do see the issue, will have to see what can be done

**Comment by hawkeye217** [comment, marked as answer]
Fixed in https://github.com/blakeblackshear/frigate/pull/13710

**Reply by Joeviocoe** [reply]
That was quick.

Does this only address the camera group editor?  I also note that it was the same issue for Exporting, if you want to add a Name, it has the same issue.

**Reply by hawkeye217** [reply]
Yes, this is for camera groups only. We'll look into the Export name separately.

**Reply by Joeviocoe** [reply]
Okay, thanks.
These are the only two UI form "text input" parts I've seen since I started looking at the Beta yesterday.  I don't know if there are more.
